supervisor/core/internet.py

```diff
@@ -0,0 +1,32 @@
+"""Internet object for the Supervisor."""
```

Comment:
Move the function to `host.network`. Also, add additional support for NetworkManager.Connectivity check. Some functions like docker need to know the working Host connection state. Since we use our own DNS, we can have a not working host internet but the working supervisor internet

Reply:
Hmm, getting weird if we have an 2 internet check. So let's only add sys_host.network. for the host internet check and put the Supervisor internet check into ... *I don't know* but split core into a module for that only looks also not good